### DISCUSSION TITLE
Bank Tag Layouts Overriding My Last Closed Tag Tab?

### DISCUSSION THREAD

**Discussion by CatnamedMittens** [runelite/runelite#17804] [Q&A]
Hello, 
Whenever I open and close my bank, the bank always default opens to a specific bank tag layout instead of my last opened Tag Tab like in the Bank Tag settings. So even if I am runecrafting or whatever, it opens to my Phantom Muspah bank tag. 

Any help?

**Comment by Nightfirecat** [comment]
Support for hub plugins should be directed to the author of the plugin.
You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the `?` button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the `Support` menu option.

**Reply by CatnamedMittens** [reply]
Thanks. I am not sure if this will be Bank Tags or Bank Tags Layout Plugin.
